compiler/packages/babel-plugin-react-compiler/src/Utils/TestUtils.ts

```diff
@@ -19,14 +18,24 @@ import {
   EnvironmentConfigSchema,
   PartialEnvironmentConfig,
 } from '../HIR/Environment';
+import {Err, Ok, Result} from './Result';
+import {hasOwnProperty} from './utils';
+
+function tryParseTestPragmaValue(val: string): Result<unknown, unknown> {
+  try {
+    let parsedVal: unknown;
+    const stringMatch = /^"([^"]*)"$/.exec(val);
```

Comment:
This was needed for `hookPattern` strings which contain escape character sequences